### PR TITLE
[15.0][MIG] stock_picking_operation_quick_change: Migration to 15.0

### PR DESCRIPTION
**Pull request by duyanhk15**

### CODE REVIEW

**Review by cyrilmanuel — APPROVED** [review]

**Review by yankinmax — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

File: stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# Copyright 2017 Tecnativa - Sergio Teruel
+# Copyright 2022 Tecnativa - Sergio Teruel
```
Comment (simahawk) [inline stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py:1]: Please, do not change copyright years. When only 1 year is present, is intended as "from 2017".

File: stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo.exceptions import UserError
+from odoo.tests import tagged
 from odoo.tests.common import TransactionCase
 
 
+@tagged("post_install", "-at_install")
```
Comment (simahawk) [inline stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py:9]: This test setup should have been moved to `setUpClass`.
I'll document this

**Pull request merged**